.github/CODEOWNERS

```diff
@@ -5,3 +5,4 @@
 /sample/ @Azure/aml-assets
 /data/ @Azure/azureml-data
 /models/ @azure/aml-training
+/common/ @azure/aml-training
```

Comment:
Please add the following paths.

/common/components/register_model
/common/src/model_deployment
/common/src/model_registration